application/src/main/java/com/ericsson/bss/cassandra/ecchronos/application/DefaultJmxConnectionProvider.java

```diff
@@ -50,4 +55,14 @@ public void close() throws IOException
     {
         myLocalJmxConnectionProvider.close();
     }
+
+    private String[] convert(Supplier<Security.JmxSecurity> jmxSecurity)
+    {
+        Credentials credentials = jmxSecurity.get().getCredentials();
+        if (!credentials.isEnabled())
+        {
+            return null;
```

Comment:
Doesn't this throw NPE later in LocalJmxConnectionProvider? Would it be cleaner to throw a proper descriptive exception?

Reply:
No, it just returns a "null" instead so that LocalJmxConnectionProvider knows that it shouldn't be using auth.